### PR TITLE
=doc using java.time.Duration for stream's javadsl

### PR DESCRIPTION
**Pull request by He-Pin**

### CODE REVIEW

File: akka-docs/src/test/java/jdocs/stream/GraphStageDocTest.java [hunk 1]
```diff
@@ -7,39 +7,36 @@
 import akka.Done;
 import akka.NotUsed;
 import akka.actor.ActorSystem;
-//#imports
```
Comment (hepin1989) [inline akka-docs/src/test/java/jdocs/stream/GraphStageDocTest.java]: need revert this change...

File: akka-docs/src/test/java/jdocs/stream/QuickStartDocTest.java [hunk 2]
```diff
@@ -5,27 +5,26 @@
 package jdocs.stream;
 
 //#stream-imports
-import akka.stream.*;
-import akka.stream.javadsl.*;
-//#stream-imports
```
Comment (hepin1989) [inline akka-docs/src/test/java/jdocs/stream/QuickStartDocTest.java]: do does this

File: akka-stream-tests/src/test/java/akka/stream/javadsl/SourceTest.java [hunk 3]
```diff
@@ -432,8 +432,8 @@ public void mustBeAbleToUseExpand() throws Exception {
   @Test
   public void mustProduceTicks() throws Exception {
     final TestKit probe = new TestKit(system);
-    Source<String, Cancellable> tickSource = Source.tick(FiniteDuration.create(1, TimeUnit.SECONDS),
-        FiniteDuration.create(500, TimeUnit.MILLISECONDS), "tick");
+    Source<String, Cancellable> tickSource = Source.tick(java.time.Duration.ofSeconds(1),
+            java.time.Duration.ofMillis(500), "tick");
```
Comment (raboof) [inline akka-stream-tests/src/test/java/akka/stream/javadsl/SourceTest.java]: Wouldn't it be nicer to import `java.time.Duration` and just use `Duration.ofSeconds(1)` here?

File: akka-docs/src/test/java/jdocs/stream/KillSwitchDocTest.java [hunk 4]
```diff
@@ -52,7 +52,7 @@ public void uniqueKillSwitchShutdownExample() throws Exception {
     //#unique-shutdown
     final Source<Integer, NotUsed> countingSrc =
       Source.from(new ArrayList<>(Arrays.asList(1, 2, 3, 4)))
-        .delay(FiniteDuration.apply(1, TimeUnit.SECONDS), DelayOverflowStrategy.backpressure());
+        .delay(      java.time.Duration.ofSeconds(1), DelayOverflowStrategy.backpressure());
```
Comment (raboof) [inline akka-docs/src/test/java/jdocs/stream/KillSwitchDocTest.java]: Spurious spaces

**Review by raboof — COMMENTED** [review]
I like the general idea, but it seems there are some whitespace problems and it'd be nicer to import `java.time.Duration` instead of spelling it out.

File: akka-docs/src/test/java/jdocs/stream/KillSwitchDocTest.java (hunk 4 above)
Comment (hepin1989) [reply akka-docs/src/test/java/jdocs/stream/KillSwitchDocTest.java]: We need a Java formatting tool which hooked into SBT:)

Comment (raboof) [reply akka-docs/src/test/java/jdocs/stream/KillSwitchDocTest.java]: Yes #20891 :)

**Review by patriknw — APPROVED** [review]
LGTM, after whitespace cleanup

File: akka-docs/src/test/java/jdocs/stream/StreamBuffersRateDocTest.java [hunk 5]
```diff
@@ -77,12 +75,12 @@ public void demonstrateBufferSizes() {
   @Test
   public void demonstrateBufferAbstractionLeak() {
     //#buffering-abstraction-leak
-    final FiniteDuration oneSecond =
-        FiniteDuration.create(1, TimeUnit.SECONDS);
+    final Duration oneSecond =
+        Duration.ofSeconds(1);
```
[the remark this answers is not in the document]
Comment (hepin1989) [reply akka-docs/src/test/java/jdocs/stream/StreamBuffersRateDocTest.java]: Really need the Java formatting tool:)

**Comment by patriknw** [comment]
Refs https://github.com/akka/akka/issues/24646

**Review by patriknw — APPROVED** [review]
LGTM, thanks
